docs/ref/settings.txt

```diff
@@ -403,6 +403,13 @@ Storing the CSRF token in a cookie (Django's default) is safe, but storing it
 in the session is common practice in other web frameworks and therefore
 sometimes demanded by security auditors.
 
+.. note::
```

Comment:
Prefer `..admonition: descriptive title` to `..note::` but I think a separate paragraph is fine. Use callout boxes sparingly.